src/measure_theory/vector_measure.lean

```diff
@@ -427,4 +427,39 @@ end
 
 end measure
 
+namespace vector_measure
+
+variables {M : Type*} [topological_space M] [ordered_add_comm_monoid M]
+
+instance : partial_order (vector_measure α M) :=
```

Comment:
This instance doesn't need `ordered_add_comm_monoid`, right? Just a partial order and unrelated additive structure will do

Reply:
Should I do the same for the `covariant_class` instance as well? i.e. decompose `ordered_add_comm_monoid` into `add_comm_monoid` + `partial_order` + `covariant_class M M (+) (\le)`?

Reply:
That probably makes sense, yes

Reply:
Done!